### PR TITLE
🚑 done some stuff

### PR DESCRIPTION
**Pull request by ashirbad29**
🌵🌵🌵🧑‍💻

### CODE REVIEW

**Comment by ashirbad29** [comment]
(⌐■_■)

File: src/Navbar.tsx [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+              Ember
+            </span>
+          </a>
+          <div className="flex items-center">
+            <a
+              href="https://github.com/Team-Asyncc/ember"
+              className="text-sm font-medium text-blue-600 hover:underline dark:text-blue-500">
+              Github
+            </a>
+          </div>
```
Comment (ayush-pandey007) [inline src/Navbar.tsx:19]: noice

File: src/components/ChatMessage.tsx [hunk 2]
```diff
@@ -1,21 +1,28 @@
...
+type PropTypes = {
+  message: {
+    body: string;
+    createdAt: Date;
+    userName: string;
+  };
+};
+
+function ChatMessage(props: PropTypes) {
+  const { body, userName } = props.message;
```
Comment (ayush-pandey007) [inline src/components/ChatMessage.tsx:12]: LGTM

File: src/components/ChatMessage.tsx [hunk 3]
```diff
@@ -1,21 +1,28 @@
...
+      <div>
+        <div className="flex space-x-2 py-2">
+          <img
+            className="h-6 w-8"
+            src={`https://avatars.dicebear.com/api/${getKeyword()}/cat.svg`}
+            referrerPolicy="no-referrer"
+          />
+          <span className="text-red-600">{userName || 'noob'}</span>
+        </div>
+        <p>{body}</p>
```
Comment (ayush-pandey007) [inline src/components/ChatMessage.tsx:25]: FINE

**Review by ayush-pandey007 — APPROVED** [review]

**Review by signifershrey — APPROVED** [review]
LGTM 😁

**Pull request merged**